search/src/main/res/layout/search_app_bar_layout.xml

```diff
@@ -21,17 +26,16 @@
         android:id="@+id/searchEditText"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/marginStandard"
         android:background="@android:color/transparent"
         android:hint="@string/looking_for_artwork"
         android:lines="1"
         android:maxLines="1"
-        android:padding="@dimen/marginStandard"
         android:textColor="@color/white"
         android:textColorHint="@color/hintColor"
         android:imeOptions="actionDone"
         android:inputType="text"
-
+        android:textSize="17sp"
+        android:fontFamily="@font/ideal_sans_medium"
```

Comment:
Please add an `android:fontFeatureSettings="tnum"` too, so numbers show up in the approved way.